Automattic/simplenote-android
# Update Welcome Note

### Fix
Update the welcome note text with new searching and pinning instructions.  See the table below for the welcome note content before and after these changes.

<table>
<tr>
<th>
Before
</th>
<th>
After
</th>
</tr>
<tr>
<td width="50%" valign="top">
Welcome to Simplenote Android!<br>Open this for instructions.<br><br>To create a new note, tap the blue Add Note button.<br><br>To search your notes, tap the search button in the Action Bar and enter any text. Simplenote will show you matching results instantly.<br><br>Got a really important note? Press the pin button while viewing a note to pin it to the top of the list.<br><br>Use tags to help organize your notes. You can also add an email address as a tag to share a note with someone.<br><br>Easily share notes with other Android apps by tapping the share button in the toolbar.<br><br>Deleted notes go in the trash. You can restore them if you want, or empty the trash to get rid of them forever.<br><br>You can access your notes on the web and your other devices. Go to http://simplenote.com to get started.<br><br>We hope you enjoy using Simplenote!
</td>
<td width="50%" valign="top">
Welcome to Simplenote Android!<br>Open this note for instructions.<br><br>To create a new note, tap the blue "New Note" button with the plus icon.<br><br>To search your notes, tap the "Search" button with the magnifying glass icon and enter any text. Simplenote will show you recent searches initially and suggested searches while entering text. Notes matching the entered text will be shown once the search is submitted.<br><br>Got a really important note? Pin it to the top of the note list. You can pin a note in two ways. Long-press the note in the list and tap the "Pin to top" button with the pin icon. Tap the "Info" button with the "i" icon while viewing a note and turn the "Pin to top" switch on.<br><br>Use tags to help organize your notes. Create a tag in the "Add tag..." field at the bottom of a note. You can also add an email address as a tag to share a note with someone.<br><br>Easily share notes with other Android apps by tapping the "Share" button with the share icon while viewing a note.<br><br>Deleted notes go in the trash. You can restore them if you want, or empty the trash to get rid of them forever.<br><br>You can access your notes on the web and your other devices. Go to http://simplenote.com to get started.<br><br>We hope you enjoy using Simplenote!
</td>
</tr>
</table>

### Test
The welcome note is only shown once per device.  It can be triggered by clearing the app data.  To be sure it is created, uninstalling and reinstalling the app is suggested.
1. Clear app data.
2. Uninstall app.
3. Install app.
4. Log in.
5. Notice ***Welcome to Simplenote Android!*** note in list.
6. Tap ***Welcome to Simplenote Android!*** note in list.
7. Notice content shown above.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
